corehq/apps/hqwebapp/tests/test_requirejs.py

```diff
@@ -77,15 +87,16 @@ def test_requirejs_disallows_hqimport(self):
                       and not f.endswith("hqwebapp/js/knockout_bindings.ko.js")]
 
         def _test_file(filename):
-            proc = subprocess.Popen(["grep", "hqImport", filename], stdout=subprocess.PIPE)
-            (out, err) = proc.communicate()
-            for line in [b.decode('utf-8') for b in out.split(b"\n")]:
-                if line:
-                    match = re.search(r'hqImport\([\'"]([^\'"]*)[\'"]', line)
-                    if match:
-                        errors.append("{} imported in {}".format(match.group(1), filename))
-                    else:
-                        errors.append("hqImport found in {}: {}".format(filename, line.strip()))
+            with open(filename, 'r') as f:
+                line = f.readline()
+                while line:
```

Comment:
nit: use of `f.readline()` (also below on line 99) and uncompiled regexes. Also, literal string search is way faster than a regex.

Suggested changes:
```py
hqImport = re.compile(r'hqImport\([\'"]([^\'"]*)[\'"]')
for line in f:
    if 'hqImport' in line:
        match = hqImport.search(line)
```

Reply:
dd4f9d760ca